Kajabi/sage-lib
# fix(colors): update tokens to match color palette updates

## Description

Update `sage_tokens.rb` to align with `_color_palette.scss`. 

## Testing in `sage-lib`

1. Navigate to designs https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/Sage-3-for-Admin?node-id=3554%3A25
2. Navigate to http://localhost:4000/pages/foundations/color
3. Compare and confirm colors match designs

## Testing in `kajabi-products`
1. (**LOW**) Color update in tokens file.

## Related
Closes #1035
